src/Api.elm

```diff
@@ -74,6 +76,15 @@ opprettCv msgConstructor =
         }
 
 
+logError : (Result Error () -> msg) -> Feilmelding -> Cmd msg
+logError msgConstructor feilmelding =
+    Http.post
+        { url = "/cv-samtale/log"
+        , expect = expectWhatever msgConstructor
+        , body = Feilmelding.encode feilmelding |> jsonBody
```

Comment:
Dette er bare en liten ting, men kanskje du skal skrive dette på 3 linjer i stedet for én?
```elm
, body =  feilmelding
              |> Feilmelding.encode
              |> jsonBody
```
Det er opp til deg, men jeg fortrekker i hvert fall å ha ting på flere linjer når jeg piper. Og her gjør du to operasjoner, først encoding, og så lager en json body, så da tenker jeg at man ser litt fortere hva som skjer hvis det er spredd på flere linjer.